examples/webserver-py/Pulumi.yaml

```diff
@@ -1,3 +1,6 @@
 name: webserver-py
-runtime: python
+runtime:
+  name: python
+  options:
```

Comment:
This is required based on https://github.com/pulumi/verify-provider-release?tab=readme-ov-file#python